services/ui-src/src/utils/reports/entities.ts

```diff
@@ -15,6 +20,56 @@ const getCheckboxValues = (entity: EntityShape | undefined, label: string) => {
   );
 };
 
+const getReportingRateType = (entity: EntityShape | undefined) => {
+  return entity?.qualityMeasure_reportingRateType?.[0]?.value ===
+    "Cross-program rate"
+    ? "Cross-program rate: " +
+        entity?.qualityMeasure_crossProgramReportingRateProgramList
+    : entity?.qualityMeasure_reportingRateType?.[0]?.value;
+};
+
+const getReportingPeriod = (entity: EntityShape | undefined) => {
+  return entity?.qualityMeasure_reportingPeriod?.[0]?.value === "No"
+    ? `${entity?.qualityMeasure_reportingPeriodStartDate} - ${entity?.qualityMeasure_reportingPeriodEndDate}`
+    : entity?.qualityMeasure_reportingPeriod?.[0]?.value;
+};
```

Comment:
Note: I looked long and hard at these two trying to think of a way to reliably abstract away the need for them, and I have an idea or two, but for right now it doesn't make sense to mess with it. Something to think on in the future, though. 👍